runtime/mangata-rococo/src/migration.rs

```diff
@@ -0,0 +1,67 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+use super::*;
+use frame_support::{
+	storage::{
+		migration::{move_prefix, storage_key_iter},
+		storage_prefix, unhashed,
+	},
+	traits::OnRuntimeUpgrade,
+	StoragePrefixedMap, Twox64Concat,
+};
+use log::info;
+use sp_runtime::traits::Zero;
+use xcm::IntoVersion;
+
+pub struct AssetRegistryMigration;
```

Comment:
[optional] ideally AssetRegistryMigration should be put in `common_runtime`. Then you will have no access to `Runtime` directly but you can make AssetRegistryMigration generic over Runtime with proper bounds

https://github.com/mangata-finance/mangata-node/blob/3a395521bf47d15d6eeb669c36df1e1c7a8d6e8a/runtime/common/src/lib.rs#L457-L468